schema/swagger-extensions.json

```diff
@@ -1589,6 +1592,51 @@
         "enum": ["create", "read", "update"]
       },
       "uniqueItems": true
+    },
+    "xmsResponseExample": {
+      "type": "object",
+      "description": "Provides the format of specifying an example response in the x-ms-examples extension.",
+      "properties": {
+        "headers": {
+          "type": "object",
+          "description": "Examples of response headers like 'location', 'azure-asyncoperation', 'retry-after', etc.",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "body":{
+          "type": ["object", "number", "array", "integer", "string", "boolean", "null"],
```

Comment:
Is specifying the type necessary? It seems to not actually be restricting the type in practice.